signbank/dictionary/forms.py

```diff
@@ -102,6 +102,9 @@ class GlossSearchForm(forms.ModelForm):
     hasnovideo = forms.BooleanField(label=_('No videos'), required=False)
     multiplevideos = forms.BooleanField(label=_('Multiple videos'), required=False)
 
+    location = forms.ModelChoiceField(label=_('Location'), queryset=FieldChoice.objects.filter(field='location'),
```

Comment:
If you specify [`to_field_name='machine_value'`](https://docs.djangoproject.com/en/4.0/ref/forms/fields/#django.forms.ModelChoiceField.to_field_name) here, you won't need to look up the field choice above, you can just pass it straight to your queryset